template.php

```diff
@@ -86,7 +87,14 @@ function uikit_base_preprocess_block(&$variables) {
 
   // Add some classes to the block title and content wrapper
   $variables['title_attributes_array']['class'] = 'block__title';
-  $variables['content_attributes_array']['class'] = 'block__content content';
+  $variables['content_attributes_array']['class'] = array('block__content', 'content');
+
+  // Global footer links class
+  if($variables['block']->region == 'footer_bottom'){
+    $variables['content_attributes_array']['class'][] = 'footer-links';
```

Comment:
This will apply this class to any block in the footer, suggest that it's targeted at menu blocks.  If you check the code a bit further down, there's already most of the logic you'd need to do that - this could possibly be incorporated in to that code.